microsetta_interface/templates/sample.jinja2

```diff
@@ -134,7 +134,7 @@
           <div class="form-group">
             <label for="sample_notes" id="sample_notes_label">Notes</label>
             <div class="col-sm-10">
-                <textarea class="form-control" id="sample_notes" name="sample_notes" rows=3>{{ sample.sample_notes |e}}</textarea>
+                <textarea class="form-control" id="sample_notes" name="sample_notes" placeholder="Is there any additional information you think we should know about this sample?  Time of day, special diet, recent antibiotics, collection issues, etc." rows=3>{{ sample.sample_notes if sample.sample_notes is not none else '' |e}}</textarea>
```

Comment:
Time of day would be redundant with the collection time stamp. Recent abx is part of the survey. What about just "(optional) Is there else about this sample that you would like to add?"